.github/workflows/deploy-web4-contract.yml

```diff
@@ -23,7 +23,8 @@ jobs:
         run: echo "POSTHOG_API_KEY=${{ secrets.POSTHOG_API_KEY }}" >> $GITHUB_ENV
       - name: Deploy to ${{ matrix.environment }}
         run: |
-          cargo near deploy --no-docker "${{ vars.NEAR_SOCIAL_ACCOUNT_ID }}" \
+          cargo build # TODO: Remove this step once https://github.com/near/cargo-near/issues/287 is fixed
```

Comment:
It was resolved and the fix is released: https://github.com/near/cargo-near/releases/tag/cargo-near-v0.13.3